src/base/bittorrent/session.cpp

```diff
@@ -612,6 +613,17 @@ void Session::setAppendExtensionEnabled(const bool enabled)
     }
 }
 
+uint Session::stop_tracker_timeout() const
```

Comment:
function name should be in camelCase for consistency with the others.

Reply:
Also `uint -> int` in all relevant places as `settings_pack::stop_tracker_timeout` is int.